lib/src/widgets/apple_pay.dart

```diff
@@ -8,26 +9,52 @@ import 'package:moyasar/src/models/payment_request.dart';
 import 'package:moyasar/src/models/sources/apple_pay/apple_pay_request_source.dart';
 
 /// The widget that shows the Apple Pay button.
-class ApplePay extends StatelessWidget {
-  const ApplePay(
-      {super.key, required this.config, required this.onPaymentResult});
+class ApplePay extends StatefulWidget {
+  const ApplePay({
+    super.key,
+    required this.config,
+    required this.onPaymentResult,
+    this.onPressed,
+  });
 
   final PaymentConfig config;
   final Function onPaymentResult;
+  final VoidCallback? onPressed;
+
+  @override
+  State<ApplePay> createState() => _ApplePayState();
+}
+
+class _ApplePayState extends State<ApplePay> {
+  String _merchantName = "";
+
+  @override
+  initState() {
```

Comment:
`void` missing

Reply:
Still not fixed

Reply:
Still not fixed

Reply:
You need to make it like this `void initState` because for now, it is just `initState`